# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/VideoTrack.cpp
# Commit: c54cb3c10992
# Full Hash: c54cb3c109922bc15fb6d0ca67e5f2a05980c1a5
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-07-31 15:46:43
# Regressor Bug: 1493613
# File Overlap Count: 2
# Description:
#   Bug 1493613 - Move MediaStream control from DOMMediaStream to MediaStreamTrack. r=padenot
#   
#   This is inherently large, because modifying these bits of DOMMediaStream and
#   MediaStreamTrack affects all consumers and producers of all DOMMediaStreams and
#   MediaStreamTracks.
# ==============================================================================

diff -r 0b03dd9d20ac -r c54cb3c10992 dom/media/VideoTrack.cpp
--- a/dom/media/VideoTrack.cpp	Wed Jul 31 09:35:06 2019 +0000
+++ b/dom/media/VideoTrack.cpp	Wed Jul 31 07:58:17 2019 +0000
@@ -16,12 +16,12 @@
 VideoTrack::VideoTrack(nsIGlobalObject* aOwnerGlobal, const nsAString& aId,
                        const nsAString& aKind, const nsAString& aLabel,
                        const nsAString& aLanguage,
-                       VideoStreamTrack* aStreamTarck)
+                       VideoStreamTrack* aStreamTrack)
     : MediaTrack(aOwnerGlobal, aId, aKind, aLabel, aLanguage),
       mSelected(false),
-      mVideoStreamTrack(aStreamTarck) {}
+      mVideoStreamTrack(aStreamTrack) {}
 
-VideoTrack::~VideoTrack() {}
+VideoTrack::~VideoTrack() = default;
 
 NS_IMPL_CYCLE_COLLECTION_INHERITED(VideoTrack, MediaTrack, mVideoStreamTrack)
 